packages/hw-app-eth/src/Eth.ts

```diff
@@ -202,9 +202,10 @@ export default class Eth {
     );
 
     let rlpOffset = 0;
-    let chainIdPrefix = "";
+    let chainId = 0;
```

Comment:
this must be `new BigNumber(0)` because if you don't enter the if below, `(0).time(2)` is going to badly break.

TypeScript spotted it as the CI is failing right now.

Reply:
done :)